subprojects/packagefiles/cmph/tests/workaround/stdlib_pch.h

```diff
@@ -0,0 +1,3 @@
+// HACK this file is used for creating precompiled header which adds missing #include <stdlib.h> that
+// prevents building the tests with MSVC
+#include <stdlib.h>
```

Comment:
I could do that, but this would make the fix non-portable; I mean, the original sources lack #include <stdlib.h> that is required to get malloc()/free() on all platforms, not only on Windows. It's just a coincidence that on non-Windows platforms those functions are included transitively via other headers. I think the use of PCH for this increases the chances this will also build on other platforms that I haven't tested with. I guess this might be the case also with Windows clang, which would probably require some clang-ish replacement of /FI because it's using the same CRT as MSVC (I tested it with clang only after fixing issues with MSVC first). And Windows clang is exactly the compiler I care about the most.

Reply:
Actually, could you not put this in `config.h`?

Another notable disadvantage with the current approach is that if you compile without PCH it will fail for unintuitive reasons.